modules/oc-mirror-dry-run-v2.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+// * microshift_running_apps/microshift_operators/microshift-operators-olm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oc-mirror-dry-run-v2_{context}"]
+= Performing a dry run for V2
+
+You can use oc-mirror to perform a dry run, without actually mirroring any images. This allows you to review the list of images that would be mirrored. A dry run also allows you to catch any errors with your image set configuration early. 
+
+.Procedure
+
+. Run the `oc mirror` command with the `--dry-run` flag to perform a dry run:
++
+[source,terminal]
+----
+$ oc mirror --config <imageset-Config.yaml> <1>
+file://<oc-mirror workspace path> <2>
+--dry-run <3>
+----
+<1> Pass in the image set configuration file that was created. This procedure assumes that it is named `imageset-config.yaml`.
+<2> Specify the oc-mirror workspace path.
+<3> Use the `--dry-run` flag to generate the dry run artifacts.
+
+.Verification
+
+. Navigate into the workspace directory that was generated:
++
+[source,terminal]
+----
+$ cd <oc-mirror workspace path>
+----
+
+. Review the `mapping.txt` file that was generated. This file contains a list of all images that would be mirrored.
+
+
+["oc-mirror-dry-run-workflow"]
+== Use-case scenarios for dry run
+
+.Mirror to Disk
+
+* Use the following command:
++
+[source,terminal]
+----
+$ oc mirror -c <image_set_config.yaml> file://<oc-mirror workspace path> --dry-run --v2
+----
++
+[NOTE]
+====
+When the image set configuration file contains images that are not present in the oc-mirror cache, these images will be added to the missing.txt file. This file can be found in the dry-run folder.
+====
+
+.Disk to Mirror
+
+* Use the following command:
+
+[source,terminal]
+----
+$ oc mirror -c <image_set_config.yaml> --from <oc-mirror workspace path> docker://<mirror-registry-url> --dry-run --v2
```

Comment:
Better to give accurate example format :
`oc mirror -c <image_set_config.yaml> --from file://<oc-mirror workspace path> docker://<mirror-registry-url> --dry-run --v2`